system_upgrade.py

```diff
@@ -125,6 +126,20 @@ def checkDNFVer():
     if DNFVERSION < StrictVersion("1.1.0"):
         raise CliError(_("This plugin requires DNF 1.1.0 or later."))
 
+def enable_blanking(arg=10):
+    try:
+        tty = open('/dev/tty0', 'wb')
+    except Exception:
+        return
+    tty.write(b'\33[9;' + str(arg).encode('ascii') + b']')
```

Comment:
It think it would be safer if this command was inside the try statement as well (in case there's some exception related to that)